Sources/Extensions/Widgets/Actions/WidgetActions.swift

```diff
@@ -32,7 +32,7 @@ struct WidgetActions: Widget {
         .contentMarginsDisabledIfAvailable()
         .configurationDisplayName(L10n.Widgets.Actions.title)
         .description(L10n.Widgets.Actions.description)
-        .supportedFamilies(WidgetActionSupportedFamilies.families)
+        .supportedFamilies([.systemSmall, .systemMedium, .systemLarge, .systemExtraLarge, .accessoryCircular])
```

Comment:
Can you keep the enum and than make 2 variables, one for legacy action widgets families and one for the latest action widgets?
And more for other widgets if needed of course